lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -217,42 +217,34 @@ def test_requested_fields_param(self):
         )
         self.verify_response_with_requested_fields(response)
 
+    def test_with_list_field_url(self):
+        url = '{0}?{1}&requested_fields={2}&requested_fields={3}&requested_fields={4}'.format(
+            self.url,
+            '&'.join(['{0}={1}'.format(param, value) for param, value in self.query_params.iteritems()]),
+            self.requested_fields[0],
+            self.requested_fields[1],
+            join(self.requested_fields[1:], ','),
+        )
```

Comment:
Nit: Consider using [urlparse.urlunparse](https://docs.python.org/2/library/urlparse.html#urlparse.urlunparse) instead of more string joining.  (Handles edge cases better and documents intent)